core/backend/src/workspace/Workspace.ts

```diff
@@ -308,7 +308,7 @@ export class ITwinWorkspace implements Workspace {
 
   public constructor(settings: Settings, opts?: WorkspaceOpts) {
     this.settings = settings;
-    this.containerDir = opts?.containerDir ?? join(NativeLibrary.defaultLocalDir, "iTwin", "Workspace");
+    this.containerDir = opts?.containerDir ?? join(IModelHost.cacheDir, "Workspace");
```

Comment:
leaving this comment to be unresolved because I do not intend to complete this PR. I just want to prove this change fixed previous errors

Reply:
core-backend still failing (crashing?) on Windows.
At beginning of tests (as in many other tests):
```
  Category
Error | GeoCoord | Error: create CloudCache error: BE_SQLITE_BUSY, account=https://geocoord-workspace.itwinjs.org/ 
```
At end of tests:
```
2022-07-14T17:16:44.7211393Z [0m  GeoServices[0m
2022-07-14T17:16:48.0330818Z   [32m  √[0m[90m should be able to interpret to completion an incomplete GeographicCRS[0m[31m (3213ms)[0m
2022-07-14T17:16:56.7477638Z 
2022-07-14T17:16:56.7479159Z =============================== Coverage summary ===============================
2022-07-14T17:16:56.7480194Z Statements   : Unknown% ( 0/0 )
2022-07-14T17:16:56.7481833Z Branches     : Unknown% ( 0/0 )
2022-07-14T17:16:56.7482650Z Functions    : Unknown% ( 0/0 )
2022-07-14T17:16:56.7483225Z Lines        : Unknown% ( 0/0 )
2022-07-14T17:16:56.7483902Z ================================================================================
2022-07-14T17:16:56.8010919Z Returned error code: 3221225725
2022-07-14T17:16:56.8012076Z [31m"@itwin/core-backend" failed to build.[39m
```

Reply:
This is now a much bigger PR, and I do intend to complete it. It fixes the full-stack-backend, transformer, and presentation tests to use their own cacheDir, which has always been a problem but is now more severe because workspaces use it when you open an iModel.